manager/controllers/baseline_update.go

```diff
@@ -0,0 +1,155 @@
+package controllers
+
+import (
+	"app/base"
+	"app/base/database"
+	"app/base/models"
+	"app/manager/middlewares"
+	"encoding/json"
+	"net/http"
+	"strconv"
+
+	"github.com/gin-gonic/gin"
+	"github.com/pkg/errors"
+	"gorm.io/gorm"
+)
+
+type UpdateBaselineRequest struct {
+	Name         string          `json:"name"`
+	InventoryIDs map[string]bool `json:"inventory_ids"`
+	Config       BaselineConfig  `json:"config"`
+}
+
+type BaselineConfig struct {
+	ToTime string `json:"to_time"`
+}
```

Comment:
There's already BaselineConfig struct in database/baseline.go
```
type BaselineConfig struct {
	ToTime time.Time `json:"to_time"`
}
```
it should be probably unified.

Reply:
Fixed.